packages/sdk-browser/src/sdk-client.ts

```diff
@@ -159,28 +159,21 @@ export class TurnkeyBrowserSDK {
 
   logoutUser = async (): Promise<boolean> => {
     await removeStorageValue(StorageKeys.CurrentUser);
-    await removeStorageValue(StorageKeys.AuthBundle);
```

Comment:
I was thinking we could leave these getters/deletions in for now, but remove the _creation path_ for `AuthBundle`s (such that if you have an `AuthBundle`, things will still work, but any time you create a new session, it'll create a `Session`)

But now that I think about it, this is fine too, especially as it'll simplify our logic on the react context side of things. Let's just be sure to make note of this change in the changelog

Reply:
addressed